src/main/java/seedu/address/logic/commands/AddMeetingCommand.java

```diff
@@ -89,8 +89,8 @@ public CommandResult execute(Model model) throws CommandException {
     }
 
     private String generateSuccessMessage(Person personToEdit) {
-        String message = !(meeting.meetingName.meetingName.isBlank() || meeting.venue.value.isBlank()
-                || meeting.when.value == null)
+        String message = !(meeting.isMeetingNameBlank() || meeting.isVenueBlank()
+                || meeting.isWhenEmpty())
                 ? MESSAGE_ADD_MEETING_SUCCESS : MESSAGE_ADD_MEETING_FAILURE;
         return String.format(message, Messages.format(personToEdit));
```

Comment:
```suggestion
        String successMessage = String.format(MESSAGE_ADD_MEETING_SUCCESS , Messages.format(personToEdit));
        String failureMessage = String.format(MESSAGE_ADD_MEETING_FAILURE, Messages.format(personToEdit));
        
        if (meeting.isMeetingNameBlank()) {
            return failureMessage;
        }
        
        if (meeting.isVenueBlank()) {
            return failureMessage;
        }
        
        if (meeting.isWhenEmpty()) {
            return failureMessage;
        }
        
        return successMessage ;
```

I feel like separating it into 3 guard clauses this way will help to improve readability and debugging.

Reply:
Thank you for pointing it out, will get to resolving this